src/components/StabilityFeeAlert.jsx

```diff
@@ -0,0 +1,34 @@
+// Libraries
+import React from "react";
+import {inject, observer} from "mobx-react";
+
+// Components
+import InlineNotification from "./InlineNotification";
+
+@inject("content")
+@inject("network")
+@observer
+class StabilityFeeAlert extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = { show: true }
+  }
+
+  render() {
+    return (
+      this.props.content.showStabilityFeeAlert() &&
+        this.state.show &&
+        !localStorage.getItem(`StabilityFeeChangeAlertClosed-${this.props.content.stabilityFeeMarkdown()}`) &&
+        this.props.network.isConnected &&
+        this.props.network.defaultAccount &&
```

Comment:
Do we really need to check for these two things?

I'm guessing if user is in the dashboard, they are connected, and I don't see why the type of account matters for showing this or not.

Reply:
It's to make sure the notification is only displayed on the dashboard page only and not the landing page also